README.rst

```diff
@@ -42,3 +42,9 @@ reuse already existing properties file since no changes were introduced there)
 
 .. _Widgetastic: https://github.com/RedHatQE/widgetastic.core
 .. _navmazing: https://github.com/RedhatQE/navmazing/
+
+5. Build documentation
```

Comment:
Imo it shouldn't be a part of quickstart guide but rather a separate docs section, which points to readthedocs and/or suggests to build the docs locally (similar to robottelo/nailgun).
Not a blocker as the comment is a bit out of scope (there's #21 to rework readme file).

Reply:
Fixed